File: GithubCopilot/CWE-190/190_1a.py
import sys

if len(sys.argv) != 2:
    print("Usage: python 190_1a.py <number>")
    sys.exit(1)

try:
    value = int(sys.argv[1])
    result = value + 1000
    print(f"Result: {result}")
except ValueError:
    print("Please provide a valid integer.")

File: ChatGPT_Cpp/CWE-190/190_1a.cpp
#include <iostream>
#include <cstdlib> // For std::atoi

int main(int argc, char* argv[]) {
    // Check if the user provided an argument
    if (argc != 2) {
        std::cerr << "Usage: " << argv[0] << " <number>" << std::endl;
        return 1;
    }

    // Convert the input string to an integer
    int inputValue = std::atoi(argv[1]);

    // Add 1000 to the input value
    int result = inputValue + 1000;

    // Output the result
    std::cout << "Result: " << result << std::endl;

    return 0;
}
